src/main/java/org/team1540/bunnybotTank2023/commands/drivetrain/TankdriveCommand.java

```diff
@@ -19,8 +19,8 @@ public TankdriveCommand(Drivetrain drivetrain, CommandXboxController xBoxControl
     }
 
     public void execute() {
-        double leftInput = xBoxController.getLeftY();
-        double rightInput = xBoxController.getRightY();
+        double leftInput = -xBoxController.getLeftY();
+        double rightInput = -xBoxController.getRightY();
```

Comment:
Why are you making these negative instead of inverting the motors? Presumably if your controller inputs need to be flipped, so would autos

Reply:
These are simulation changes, I was messing around with teleop controls and noticed that they were inverted. Autos work fine, so maybe the controller is inverted?

Reply:
I just tested the controller on the actual DS app, the joysticks axes are in fact inverted, although this might be a skill issue with my controller since it's an xbox 1 controller vs an xbox series controller

Reply:
So it sounds like your controller is just weird then, you might be able to invert in the simulator gui, but you should flip this back before merging.